app/controllers/searches_controller.rb

```diff
@@ -5,18 +5,55 @@ class SearchesController < CatalogController
 
   def index
     @search = FileSearch.new(params, catalog_query: self)
-    @results = @search.results
+    get_audios
```

Comment:
I don't have a problem with fetch_. There's a good chance this controller will get changed quite a bit before we're done.